src/components/Testimonials/Testimonials.js

```diff
@@ -62,3 +70,25 @@ Testimonials.defaultProps = {
 }
 
 export default withStyles(s)(Testimonials)
+
+function CustomNextArrow(props) {
```

Comment:
This is a stateless component. use the same format as `Aavatar.js`, look how I'm declaring the component there, and how I use `Props`

Reply:
Now I see that in `TestimonialItems` you are using the correct format.

Why are you using the `function` here instead of the same way you did `TestimonialItems` ?